index.html

```diff
@@ -862,9 +862,18 @@
         <section>
             <h3>Implicit Release of Pointer Capture</h3>
             <p>Immediately after firing the <code>pointerup</code> or <code>pointercancel</code> events, a user agent MUST clear the <a>pending pointer capture target override</a> for the <code>pointerId</code> of the <code>pointerup</code> or <code>pointercancel</code> event that was just dispatched, and then run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> steps to fire <code>lostpointercapture</code> if necessary.</p>
+            <p>Immediately before <a href="https://html.spec.whatwg.org/multipage/dnd.html#initiate-the-drag-and-drop-operation">drag operation starts</a> [[!HTML]],
+a user agent MUST send a <code>pointercancel</code> for the pointer that caused the drag operation.
+This <code>pointercancel</code> will follow the paragraph above regarding releasing the capture.</p>
```

Comment:
Sorry for not noticing this before: can't we move this paragraph to the Section 5.2.6 (`pointercancel`) above?  Then the second sentence here won't be needed because it's implied.  Moreover, the PR here is more about pointercancel than about implicit release.